iot_verifier/Cargo.toml

```diff
@@ -55,3 +55,4 @@ price = { path = "../price" }
 tokio-util = { workspace = true }
 tokio-stream = { workspace = true }
 task-manager = { path = "../task_manager" }
+time = "0.3.17"
```

Comment:
We don't usually pin deps to a specific version in the Cargo.toml, is this exact version needed?

Reply:
actually this is not even required...will remove